portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Fetches stats from the servers and adds them to the DOM.
+ */
+function generateDiscussion() {
+  	console.log('Fetching discussion.');
+
+    // fetch() function returns a promise
+    const discussionLogPromise = fetch('/data');
+
+	// When the request is complete, pass the response into parseJSON()
+    discussionLogPromise.then(parseJSON);
+}
+
+
+// This function will handle the response and turn it to json
+function parseJSON(response){
+    console.log('Parse the response into JSON');
+
+    const discussionLog = response.json();
+
+    discussionLog.then(addToContainer);
```

Comment:
This function is meant to parse json, but it seems that it only converts the response to json and pass that into another function. Please rename function here to indicate that it does both of these tasks. (Include into documentation for the function as well).